app/models/post.rb

```diff
@@ -1,11 +1,24 @@
 class Post < ActiveRecord::Base
 
   include AASM
-
+  include PgSearch
   belongs_to :user
   has_many :comments
   has_many :likes
-
+  #search:
```

Comment:
@babiracka same as [here](https://github.com/rishijain/oneplaceaday/pull/55/files#r36463706) – a comment doesn't add any value when it duplicates words from its method